python/cudf/cudf/dataframe/column.py

```diff
@@ -435,12 +434,14 @@ def element_indexing(self, index):
         ------
         ``IndexError`` if out-of-bound
         """
-        index = int(index)
+        index = np.int32(index)
         if index < 0:
             index = len(self) + index
         if index > len(self) - 1:
             raise IndexError
         val = self.data[index]  # this can raise IndexError
+        if isinstance(val, nvstrings.nvstrings):
+            val = val.to_host()[0]
```

Comment:
This branch should never fire. We recently fixed the bug where calling `__getitem__` on a StringColumn would return an nvstrings or StringColumn instead of a scalar string.

Reply:
Nevermind, this is moving that fix down to here. This is good.